storybook/src/Models/PermissionsModel.qml

```diff
@@ -30,55 +33,196 @@ ListModel {
+                   }
+               ])
+    }
 
     function createHoldingsModel1() {
-        return [
-            {
-                operator: OperatorsUtils.Operators.None,
-                type: HoldingTypes.Type.Asset,
-                key: "SOCKS",
-                name: "SOCKS",
-                amount: 1.2,
-                imageSource: ModelsData.assets.socks
-            },
-            {
-                operator: OperatorsUtils.Operators.Or,
-                type: HoldingTypes.Type.Asset,
-                key: "ZRX",
-                name: "ZRX",
-                amount: 15,
-                imageSource: ModelsData.assets.zrx
-            },
-            {
-                operator: OperatorsUtils.Operators.And,
-                type: HoldingTypes.Type.Collectible,
-                key: "Furbeard",
-                name: "Furbeard",
-                amount: 12,
-                imageSource: ModelsData.collectibles.kitty1
-            }
-        ]
+        var holdings = []
+        holdings.push({
+                          operator: OperatorsUtils.Operators.None,
+                          type: HoldingTypes.Type.Asset,
+                          key: "SOCKS",
+                          name: "SOCKS",
+                          amount: 1.2,
+                          imageSource: ModelsData.assets.socks,
+                          available: true
+                      });
+        holdings.push({
+                          operator: OperatorsUtils.Operators.Or,
+                          type: HoldingTypes.Type.Asset,
+                          key: "ZRX",
+                          name: "ZRX",
+                          amount: 15,
+                          imageSource: ModelsData.assets.zrx,
+                          available: false
+                      });
+        holdings.push({
+                          operator: OperatorsUtils.Operators.And,
+                          type: HoldingTypes.Type.Collectible,
+                          key: "Furbeard",
+                          name: "Furbeard",
+                          amount: 12,
+                          imageSource: ModelsData.collectibles.kitty1,
+                          available: true
+                      });
+        return holdings
     }
 
     function createHoldingsModel2() {
-        return [
-            {
-                operator: OperatorsUtils.Operators.None,
-                type: HoldingTypes.Type.Collectible,
-                key: "Happy Meow",
-                name: "Happy Meow",
-                amount: 50.25,
-                imageSource: ModelsData.collectibles.kitty3
-            },
-            {
-                operator: OperatorsUtils.Operators.And,
-                type: HoldingTypes.Type.Collectible,
-                key: "AMP",
-                name: "AMP",
-                amount: 11,
-                imageSource: ModelsData.assets.amp
-            }
-        ]
+        var holdings = []
```

Comment:
`const`. Why removing directly was wrong here? :D

Reply:
Same below, please consider removing `;`s for consistency.

Reply:
> `const`. Why removing directly was wrong here? :D

Just rebase issue. I will keep the return directly way!

Reply:
> `const`. Why removing directly was wrong here? :D

sorry, removing -> returning